docs/management/admin/admin-pg-ov.asciidoc

```diff
@@ -1,77 +1,153 @@
+
+* *Actions:* Select the context menu to do the following:
+
+** *View Host Details:* View host details on the *Hosts* page in the {security-app}.
+
+** *View Agent Policy:* View the Policy in {fleet}.
+
+** *View Agent Details:* View agent details and activity logs in {fleet}.
 
-* *Last Active:* A date and timestamp of the last time the host was active.
 
 [role="screenshot"]
 image::images/admin-pg.png[Admin page]
 
 
 *Hostname details*
 
-Click a *Hostname* link to display host details in a flyout panel. This panel also provides shortcut links to view the associated integration, view the configuration response details, and reassign the configuration if needed.
+Click a *Hostname* link to display host details in a flyout panel. This panel also provides shortcut links to view the associated Policy, view the configuration response details, and reassign the Policy if needed.
 
 [role="screenshot"]
 image::images/host-flyout.png[Admin page]
 
-*Integration details*
+*Integration Policy details*
 
-To view the Integration details page, click the link in the Integration column. If you are viewing host details, you can also click the *Integration* link on the flyout panel.
+To view the Integration Policy page, click the link in the *Integration Policy* column. If you are viewing host details, you can also click the *Integration Policy* link on the flyout panel.
 
-On this page, you can view and manage protection configuration and event collection settings. In the upper-right corner is a Key Performance Indicator (KPI) widget that provides hosts status data. If you need to update the configuration, make changes as appropriate, then click the *Save* button to save your changes.
+On this page, you can view and configure endpoint protection and event collection settings. In the upper-right corner are Key Performance Indicators (KPIs) that provide current endpoint status. If you need to update the Policy configuration, make changes as appropriate, then click the *Save* button to apply the new changes.
 
-NOTE: Users must have permission to read/write to Ingest Manager APIs to make changes to the configuration.
+NOTE: Users must have permission to read/write to Fleet APIs to make changes to the configuration.
 
 [role="screenshot"]
 image::images/integration-pg.png[Integration page]
 
 *Configuration status*
 
-The status of the configuration appears in the *Configuration Status* column and displays one of the following possibilities:
+The status of the Policy configuration appears in the *Policy Status* column and displays one of the following possibilities:
 
-* *Success:* The configuration applied successfully.
+* *Success:* The Policy applied successfully.
 
-* *Pending or Partially Applied:* The configuration is pending application, or the configuration in its entirety was not applied.
+* *Pending or Partially Applied:* The Policy is pending application, or the Policy was not applied in its entirety.
```

Comment:
there is, technically, not a pending state for the policy - you will just get a Warning or Partially Applied if there are some issues